docs/.custom_wordlist.txt

```diff
@@ -47,4 +49,4 @@ devel
 YY
 hostname
 Geneve
-Virtualization
\ No newline at end of file
+Virtualization
```

Comment:
Thank you for restoring newlines at the end of the file!

We should add a check for this in the `check-lint` step:
https://github.com/canonical/microovn/blob/4a25d8ef273bb4de70d6a7632920738d3bfeea8e/Makefile#L16-L21